Gemfile

```diff
@@ -29,7 +29,7 @@ gem "stimulus-rails"
 gem "tailwindcss-rails"
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
-gem "jbuilder"
+# gem "jbuilder"
```

Comment:
For now we are not anticipating any JSON API replies to our controller endpoints. Having this in here clutters our controllers and it causes scaffold to generate `*.jbuilder` files